packages/solo-setup/utils/transfer.ts

```
@@ -0,0 +1,185 @@
+import { ThorClient } from '@vechain/sdk-network';
+import {
+    THOR_SOLO_DEFAULT_GENESIS_ACCOUNTS,
+    THOR_SOLO_SEEDED_ACCOUNTS
```

ESLint

[error] 'THOR_SOLO_SEEDED_ACCOUNTS' is defined but never used.

(@typescript-eslint/no-unused-vars)